.github/workflows/cmake_builds.yml

```diff
@@ -134,49 +134,79 @@ jobs:
-          cd /C/tools/msys64/mingw64/share/proj
+          cd /mingw64/share/proj
           wget http://download.osgeo.org/proj/proj-datumgrid-1.8.tar.gz
           tar xvzf proj-datumgrid-1.8.tar.gz
-          '@
-          & c:\tools\msys64\usr\bin\bash.exe -lc $script
-        env:
-          MSYSTEM: MSYS
       - name: Install python dependencies for autotest
         run: |
-          $script = @'
-          python -m ensurepip --upgrade
-          python -m pip install -U wheel setuptools numpy
-          python -m pip install pytest pytest-env
-          '@
-          & c:\tools\msys64\usr\bin\bash.exe -lc $script
-        env:
-          MSYSTEM: MINGW64
+          python -m pip install pytest-env
       #  libxml2 disabled because currently causes a 'Imported target "LibXml2::LibXml2" includes non-existent path "/mingw64/include/libxml2" in its INTERFACE_INCLUDE_DIRECTORIES'
       # Disable mySQL since C:/mysql/lib/mysqlclient.lib (unrelated to msys) is found, which causes linking issues
       # Disable Python bindings because of 'ValueError: filename D:/a/gdal/gdal/swig/python/osgeo/__init__.py does not start with the input_base_dir D:/a/gdal/gdal/swig/python/osgeo/\' when running lib2to3
       # Set explicitly CMAKE_C|CXX_COMPILER otherwise C:/ProgramData/chocolatey/bin/gcc.exe would be used
       - name: Configure
-        run: cmake -DCMAKE_BUILD_TYPE=release -G "${env:generator}" -Werror=dev "-DCMAKE_C_COMPILER=c:/tools/msys64/mingw64/bin/gcc.exe" "-DCMAKE_CXX_COMPILER=c:/tools/msys64/mingw64/bin/g++.exe" "-DCMAKE_C_COMPILER_LAUNCHER=c:\tools\msys64\mingw64\bin\ccache.exe" "-DCMAKE_CXX_COMPILER_LAUNCHER=c:\tools\msys64\mingw64\bin\ccache.exe" "-DCMAKE_PREFIX_PATH=C:\tools\msys64\mingw64" "-DCMAKE_UNITY_BUILD=${env:CMAKE_UNITY_BUILD}" -S . -B "build" -DLIBXML2_INCLUDE_DIR=C:/tools/msys64/mingw64/include -DGDAL_USE_LIBXML2:BOOL=OFF -DGDAL_USE_MYSQL:BOOL=OFF -DBUILD_PYTHON_BINDINGS:BOOL=OFF -DBUILD_CSHARP_BINDINGS=ON -DCMAKE_C_FLAGS=-Werror -DCMAKE_CXX_FLAGS=-Werror -DWERROR_DEV_FLAG="-Werror=dev"
+        run: |
+          cmake -S . -B build -G "${generator}" -Werror=dev \
+            -DCMAKE_BUILD_TYPE=release \
+            -DCMAKE_C_FLAGS=-Werror \
+            -DCMAKE_CXX_FLAGS=-Werror \
+            -DWERROR_DEV_FLAG=-Werror=dev \
+            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_PREFIX_PATH=/mingw64 \
+            -DCMAKE_INSTALL_PREFIX=$PWD/install-gdal \
+            "-DCMAKE_UNITY_BUILD=${CMAKE_UNITY_BUILD}" \
+            -DLIBXML2_INCLUDE_DIR=/mingw64/include -DGDAL_USE_LIBXML2:BOOL=OFF \
+            -DGDAL_USE_MYSQL:BOOL=OFF \
+            -DGDAL_ENABLE_DRIVER_SAR_CEOS:BOOL=OFF \
+            -DOGR_ENABLE_DRIVER_MSSQLSPATIAL:BOOL=OFF \
```

Comment:
should no longer be needed with the fixes I done
```suggestion
```

Reply:
This was left over, indeed. At least it is no longer hidden in a single super long line.